src/component/MembersList.jsx

```diff
@@ -29,10 +62,20 @@ const MembersList = ({teamId}) => {
     <div className="containerList">
       <div className="titleContainer">
         <h4>Here is a list of your team members!</h4>
+        {selectedMembers.length > 0 && (
+          <button onClick={handleDeleteSelectedMembers}>Delete Selected Members</button>
+        )}
       </div>
       <div className="members-grid">
         {members.map((member) => (

```

Comment:
Are you sure 'members' has a value at all times? If not, please make sure to handle the case when 'members' is empty.